Makefile

```diff
@@ -183,5 +183,423 @@ verify_version:
+	$(COMPOSER) --working-dir=tests/Frameworks/Laravel/Version_5_7 update
+	$(call run_tests,tests/Integrations/Laravel/V5_7)
+test_web_laravel_58:
+	$(COMPOSER) --working-dir=tests/Frameworks/Laravel/Version_5_8 update
+	$(call run_tests,--testsuite=laravel-58-test)
+test_web_lumen_52:
+	$(COMPOSER) --working-dir=tests/Frameworks/Lumen/Version_5_2 update
+	$(call run_tests,tests/Integrations/Lumen/V5_2)
+test_web_lumen_56:
+	$(COMPOSER) --working-dir=tests/Frameworks/Lumen/Version_5_6 update
+	$(call run_tests,tests/Integrations/Lumen/V5_6)
+test_web_lumen_58:
+	$(COMPOSER) --working-dir=tests/Frameworks/Lumen/Version_5_8 update
+	$(call run_tests,tests/Integrations/Lumen/V5_8)
+test_web_slim_312:
+	$(COMPOSER) --working-dir=tests/Frameworks/Slim/Version_3_12 update
+	$(call run_tests,--testsuite=slim-312-test)
+test_web_symfony_23:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_2_3 update
+	$(call run_tests,tests/Integrations/Symfony/V2_3)
+test_web_symfony_28:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_2_8 update
+	$(call run_tests,tests/Integrations/Symfony/V2_8)
+test_web_symfony_30:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_3_0 update
+	php tests/Frameworks/Symfony/Version_3_0/bin/console cache:clear --no-warmup --env=prod
+	$(call run_tests,tests/Integrations/Symfony/V3_0)
+test_web_symfony_33:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_3_3 update
+	php tests/Frameworks/Symfony/Version_3_3/bin/console cache:clear --no-warmup --env=prod
+	$(call run_tests,tests/Integrations/Symfony/V3_3)
+test_web_symfony_34:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_3_4 update
+	php tests/Frameworks/Symfony/Version_3_4/bin/console cache:clear --no-warmup --env=prod
+	$(call run_tests,tests/Integrations/Symfony/V3_4)
+test_web_symfony_40:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_4_0 update
+	php tests/Frameworks/Symfony/Version_4_0/bin/console cache:clear --no-warmup --env=prod
+	$(call run_tests,tests/Integrations/Symfony/V4_0)
+test_web_symfony_42:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_4_2 update
+	php tests/Frameworks/Symfony/Version_4_2/bin/console cache:clear --no-warmup --env=prod
+	$(call run_tests,tests/Integrations/Symfony/V4_2)
+test_web_symfony_44:
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_4_4 update
+	php tests/Frameworks/Symfony/Version_4_4/bin/console cache:clear --no-warmup --env=prod
+	$(call run_tests,tests/Integrations/Symfony/V4_4)
+test_web_wordpress_48:
+	$(call run_tests,tests/Integrations/WordPress/V4_8)
+test_web_yii_2:
+	$(COMPOSER) --working-dir=tests/Frameworks/Yii/Version_2_0_26 update
+	$(call run_tests,tests/Integrations/Yii/V2_0_26)
+test_web_zend_1:
+	$(call run_tests,tests/Integrations/ZendFramework/V1)
+test_web_custom:
+	$(COMPOSER) --working-dir=tests/Frameworks/Custom/Version_Autoloaded update
+	$(call run_tests,--testsuite=custom-framework-autoloaded-test)
+
```

Comment:
It looks like `DDTrace\Tests\Integrations\Custom\NotAutoloaded\FrameworkNoAutoloadedTest` is not being run anywhere from CI. This isn't something that was introduced with this PR, but it just looks like it isn't being tested currently. Perhaps we could add the non-autoloaded custom framework to the web tests for all PHP versions?

Reply:
There was a reason it wasn't being run in CI, but I forget what it was. Only way to find out is to try enabling it, right?